data/json/items/gun/9mm.json

```diff
@@ -197,7 +197,7 @@
     "type": "GUN",
     "name": "L2032 Lookout",
     "name_plural": "L2032 Lookout",
-    "description": "After the Enforcer's success, Leadworks LLC reworked their autorevolver design to cater to the 9x19mm market.  The Lookout, released in 2032, takes advantage of the smaller 9x19mm to hold a seventh round, without sacrificing the ease of handling or reloading the Enforcer made legendary.",
+    "description": "After the Enforcer's success, Leadworks LLC reworked their autorevolver design to cater to the 9x19mm market.  The Lookout, released in 2032, takes advantage of the smaller 9x19mm to hold a seventh round, without sacrificing the ease of handling or reloading the Enforcer made legendary.  Comes with built in holographic and laser sights.  Similar to other Leadworks products it doesn't accept third-party modifications.",
```

Comment:
I suppose we need to change the dates (or remove them altogether) here and below for the same reason it was done in https://github.com/CleverRaven/Cataclysm-DDA/commit/77b874ca4ce878ce4d9fe0c7a60ac1dd48cefe6e#diff-f66af8527da2e94290d5777a157b734a.